horace_core/sqw/@sqw/symmetrise_sqw.m

```diff
@@ -68,12 +68,10 @@
 else
     proj = {line_proj([1 0 0], [0 1 0], ...
                       'alatt', win.data.proj.alatt, ...
-                      'angdeg', win.data.proj.angdeg)};
+                      'angdeg', [90, 90, 90])};
```

Comment:
as just discussed, make it clear that this is an orthogonal transformation between these two projections, not changing a projection. Also make it clear why alatt does need to be retained here.